internal/cloud/realm/services.go

```diff
@@ -0,0 +1,7 @@
+package realm
+
+// set of supported data source types
+const (
+	ClusterType  = "mongodb-atlas"
+	DataLakeType = "datalake"
```

Comment:
[nit] rename this to `ServiceTypeDatalake` (let's start to make consistent the casing of "datalake" here ... I think we're settled on it being all one word/no hyphens/no extra camel cases)